### PR TITLE
print error, but don't exit process, when in watch mode

### PR DESCRIPTION
**Pull request by bradb**
Fix watch mode to print errors, but not exit the process.

### CODE REVIEW

File: lib/bundle.js [hunk 1]
```diff
@@ -74,11 +75,11 @@ module.exports = (targetBundle, cnfg, opts) => {
     })
   }
 
-  function bundle() {
+  function bundle(errorHandler) {
```
Comment (ryanerato) [inline lib/bundle.js]: Do you meant use an error handler that's passed in or the `watchErrorHandler` imported at the top of the file?

File: lib/watchErrorHandler.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+module.exports = (message, status) =>  {
+  if(status === 0) {
+    console.log(message);
+  } else {
+    console.error( message.stack ? message.stack : String(message));
```
Comment (ryanerato) [inline lib/watchErrorHandler.js]: A couple small things:
1. Spacing
2. You could use `||` here such that `message.stack || String(message)`
3. I'm not exactly sure what `String(message)` will produce on an object. I've had good luck with `JSON.stringify(message, null, 2)` properly converting an object to a string. Oftentimes `Object.toString()` simply results in "[Object object]".

**Review by ryanerato — CHANGES_REQUESTED** [review]

File: lib/bundle.js (hunk 1 above)
Comment (bradb) [reply lib/bundle.js]: @ryanerato This arg should actually be removed. I had started implementing in a way that needed that, but then changed tack. Will remove now.

File: lib/watchErrorHandler.js (hunk 2 above)
Comment (bradb) [reply lib/watchErrorHandler.js]: @ryanerato The `String(message)` stuff is consistent with how the [hard exit handler](https://github.com/3dna/bundbi/blob/master/lib/exit.js) is written. What do you think of rolling forward with that, for consistency? (I've pushed up the `||` tweak though.)

Comment (ryanerato) [reply lib/watchErrorHandler.js]: As long as the output looks fine, I'm certainly down for consistency. I only brought it up as outputting objects in JS can be inconsistent depending on how its done.

**Review by ryanerato — APPROVED** [review]
LGTM 👍

**Pull request merged**